packages/session/src/session.ts

```diff
@@ -83,16 +96,7 @@ export function makeSession(
 }
 
 export function hashSession(session: Session): string {
-  const str = JSON.stringify(session, function (key, val) {
-    // ignore cookie property on the root object
-    if (this === session && key === 'cookie') {
-      return;
-    }
-
-    return val;
-  });
```

Comment:
This logic was left over from `express-session`, but it doesn't apply to our own code.